core/src/main/java/bisq/core/account/witness/AccountAgeWitnessService.java

```diff
@@ -880,4 +880,10 @@ public boolean isSignWitnessTrade(Trade trade) {
                 !peerHasSignedWitness(trade) &&
                 tradeAmountIsSufficient(trade.getTradeAmount());
     }
+
+    public String signInfoFromAccount(PaymentAccount paymentAccount) {
+        var pubKey = keyRing.getSignatureKeyPair().getPublic();
+        var witness = getMyWitness(paymentAccount.getPaymentAccountPayload());
+        return Utilities.bytesAsHexString(witness.getHash()) + "," + Utilities.bytesAsHexString(pubKey.getEncoded());
```

Comment:
Is that a debug util or is the "," concatenation used in normal cases?

Reply:
If debug tool better to rename method to make it more clear and add a comment...

Reply:
After reviewing the rest I understand now better... I stil would recommend to rename the method so it is more clear its not a normal user method but a special tool... also i am old-school and prefer get prefix... here it could read as intent as well: sign infoFromAccount.... instead of: get signInfo from account

Reply:
Yes, the naming was not good, changed to `getSignInfo...`